<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openAccount</class>
 <widget class="QDialog" name="openAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Idade</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Nascimento</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Tipo Conta</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="radioCorrente">
      <property name="text">
       <string>Corrente</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QRadioButton" name="radioPoupanca">
      <property name="text">
       <string>Poupança</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QRadioButton" name="radioJuridica">
      <property name="text">
       <string>Jurídica</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QLineEdit" name="EditEmail"/>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="editNascimento"/>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="editCpf"/>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="editIdade"/>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="editName"/>
    </item>
    <item row="6" column="1" colspan="3">
     <widget class="QPushButton" name="btnCreateAccount">
      <property name="text">
       <string>Criar conta</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
